dev-packages/browser-integration-tests/suites/tracing/request/fetch-with-no-active-span/init.js

```diff
@@ -5,6 +5,8 @@ window.Sentry = Sentry;
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   // disable pageload transaction
-  integrations: [Sentry.BrowserTracing({ tracingOrigins: ['http://example.com'], startTransactionOnPageLoad: false })],
+  integrations: [
+    new Sentry.BrowserTracing({ tracingOrigins: ['http://example.com'], startTransactionOnPageLoad: false }),
+  ],
```

Comment:
this test was broken before but still succeeded because our assertions didn't check anything specific other than ensuring that requests were made.